aio/content/examples/rx-library/src/retry-on-error.ts

```diff
@@ -1,20 +1,28 @@
-
-import { Observable, of } from 'rxjs';
-
-
+// #docplaster
+/*
+  Because of how the code is merged together using the doc regions,
+  we need to indent the imports with the function below.
+*/
+/* tslint:disable:no-shadowed-variable */
+/* tslint:disable:align */
 // #docregion
-
+import { of } from 'rxjs';
 import { ajax } from 'rxjs/ajax';
 import { map, retry, catchError } from 'rxjs/operators';
 
+// #enddocregion
+
+export function docRegionApiData(console, ajax) {
+// #docregion
 const apiData = ajax('/api/data').pipe(
-  retry(3), // Retry up to 3 times before failing
-  map(res => {
+  map((res: any) => {
     if (!res.response) {
+      console.log('Error occured. Retrying...');
```

Comment:
The `Retrying...` part is not necessarily true and therefore confusing.
Let's stick to:

```suggestion
      console.log('Error occured.');
```